### ISSUE TITLE
[FEAT] sort by

### ISSUE THREAD

**Issue by lvluozixuemengyanyan** [PhilippC/keepass2android#2482]
Hope to add sorting function, which can drag and sort each record up and down

**Comment by Lupusolus** [comment]
I agree! I'd love that too! At the moment, the default sorting in the app is not entirely clear to me. I assumed that this setting corresponds to the sorting in the database, but that doesn't seem to be the case. If I want to synchronise the same sorting as in the database on the computer, how is that possible?

**Comment by PhilippC** [comment]
duplicate of https://github.com/PhilippC/keepass2android/issues/1898
